README.mediawiki

```diff
@@ -1,46 +0,0 @@
-People wishing to submit QIPs, first should propose their idea to the [https://forum.qu.ai Quai forum] (do <em>not</em> assign a number - read <a href="qip-0001.mediawiki">QIP 1</a> for the full process). After discussion, please open a PR. After copy-editing and acceptance, it will be published here.
```

Comment:
In the future, it would be better to do this as a git move, that way the diff can be compared across the renamed files `git mv oldfile.name newfile.name`

Reply:
I see you did that for some of the other files. Looks like you just forgot READMEand qip-0004.

Reply:
Will need to squash after you check this out.